char/liberate/FULL_MODELS/corners/leak/discrete.spice

```diff
@@ -0,0 +1,30 @@
+* SKY130 Spice File.
```

Comment:
is that intentional to have sky130 models being referenced here?

Reply:
deleted (just for characterization)